### PR TITLE
Feature/applied old layout for new template

### PR DESCRIPTION
**Pull request by anatoliiibrahimov**
[Link to JIRA ticket](https://ssu-jira.softserveinc.com/browse/LR-194)

### Author.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [x] @OKrain 
- [x] @superNazo 
- [x] @vovka

### Issue summary.

New quiz: apply old layout for new ng template.

### Fix summary. 

Added bootstrap classes.

### CODE REVIEW

**Comment by anatoliiibrahimov** [comment]
Now it is looks :
rsz_screenshot_from_2016-04-21_224344

**Comment by OKrain** [comment]
nice :+1:

File: app/assets/javascripts/angularjs_app/components/quizzes/views/newQuiz.html [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-	<label>Content <br>
-	  <input type="text" ng-model="question.content"></input>
-	</label>
+  <div class="form-group" ng-repeat="question in quiz.questions_attributes">
+  <label> {{$index+1}} Content <br>
+    <textarea type="text" ng-model="question.content"></textarea>
+  </label>
+  </div>
+  <div class="form-group">
+  <a href="javascript:;" ng-click="add()">Add question</a>
```
Comment (superNazo) [inline app/assets/javascripts/angularjs_app/components/quizzes/views/newQuiz.html]: what does this 'href=javascript' do?

File: app/assets/javascripts/angularjs_app/components/quizzes/views/newQuiz.html [hunk 2]
```diff
@@ -1,17 +1,21 @@
 <h1>New Quiz</h1>
 
 <form ng-submit="addPost()">
```
Comment (superNazo) [inline app/assets/javascripts/angularjs_app/components/quizzes/views/newQuiz.html]: what does this 'addPost()' do? 
there is no input[type=submit], so it seems to be an excessive ng-submit directive, if it does - remove it

**Comment by superNazo** [comment]
identations are not correct - fix it

**Comment by anatoliiibrahimov** [comment]
@superNazo done

**Comment by vovka** [comment]
lgtm. 
Be careful, branch is gray and needs rebase: `git pull --rebase origin master`.

**Pull request merged**